app/editor/.devcontainer/Dockerfile

```diff
@@ -9,10 +9,10 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
   && apt-get -y install --no-install-recommends git libstdc++6
 
 # [Optional] Uncomment if you want to install an additional version of node using nvm
-ARG EXTRA_NODE_VERSION=v16.10.0
-RUN su node -c "npm config delete prefix"
-RUN su node -c "source /usr/local/share/nvm/nvm.sh && nvm install ${EXTRA_NODE_VERSION}"
-RUN su node -c "npm config set prefix $NVM_DIR/versions/node/${EXTRA_NODE_VERSION}"
+# ARG EXTRA_NODE_VERSION=v16.10.0
```

Comment:
This is causing a warning/bug in the console after the devcontainer is loaded.  I don't understand it enough to know what/why/etc.